tutorials/Python/neonUtilities/neonUtilitiesPython.md

```diff
@@ -0,0 +1,342 @@
+
+---
+syncID:63993d7274f84e51a1047e1fea9aece1
+title: "Using NEON Utilities in Python"
+description: "Use the neonUtilities R package in Python, via the rpy2 library."
+dateCreated: 2018-12-08
+authors: Claire K. Lunch
+contributors: 
+estimatedTime: 
+packagesLibraries: rpy2
+topics: data-management,rep-sci
+languagesTool: python
+dataProduct: 
+code1: /Python/neonUtilities/neonUtilitiesPython.ipynb
+tutorialSeries: 
+urlTitle: utilities-python
+---
+
```

Comment:
Future, once we figure out where training data sets will be stored, let's include a small downloadable data set that can be used for this and the neonUtilities lesson.